public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
We should look into it! Ryan already [created a palette](https://github.com/MusikAnimal/pageviews/issues/25#issuecomment-186755073) of 10 colours that are colour-blind friendly, and we have the Select2 code for it. I think it'd make the tool look really fun if the WikiProjects column looked like Fruit Loops cereal